app/controllers/people_controller.rb

```diff
@@ -0,0 +1,16 @@
+class PeopleController < ApplicationController
+  def show
+    @person = Person.find(params[:id])
+
+    render :show_people
```

Comment:
Naming convention that helps make the code flow more clearly would state that using a controller action name that matches the view name so I'd prefer "show" over "show_people"